src/components/icons/Emoji.tsx

```diff
@@ -0,0 +1,39 @@
+import path from 'node:path';
+import twemoji from '@twemoji/api';
+import type { FC } from 'react';
+
+export interface IEmoji {
+  emoji: string;
+}
+
+type TwemojiOptions = {
+  base: string;
+  size: string;
+  ext: string;
+};
+
+export const Emoji: FC<IEmoji> = (props: IEmoji) => {
+  // Generate the HTML for the emoji using twemoji
+  const emojiHtml = twemoji.parse(props.emoji, {
```

Comment:
We could also decide to store ALL emoji svgs locally instead of conditionally routing based on the callback fn below

Reply:
This is no longer a concern after pivoting to the https://github.com/discord/twemoji fork which contains SVGs with `/dist`